app/controllers/homepage_controller.rb

```diff
@@ -288,18 +289,18 @@ def search_coordinates(latlng)
     end
   end
 
-  def location_search_params(latlng, distance_unit, distance_max, minimum_distance_max, scale, limit_by_distance)
+  def location_search_params(latlng, distance_unit, distance, minimum_distance, scale_multiplier, offset_multiplier, limit_by_distance)
     # Current map doesn't react to zoom & panning, so we fetch all the results as before.
     if @view_type != 'map'
       Maybe(latlng)
         .map {
-          distance = [minimum_distance_max, distance_max.to_f].max
+          distance = [minimum_distance, distance.to_f].max
```

Comment:
Actually that was my purpose, but for clarity it's probably better to name it differently